mkt/langpacks/tests/test_utils_.py

```diff
@@ -0,0 +1,67 @@
+# -*- coding: utf-8 -*-
+from django.forms import ValidationError
+
+from mock import patch
+from nose.tools import eq_
+
+from mkt.langpacks.utils import LanguagePackParser
+from mkt.site.tests import TestCase
+
+
+class TestLangPackUpload(TestCase):
+    @patch('mkt.files.utils.WebAppParser.get_json_data')
+    def test_upload_multiple_languages(self, get_json_data_mock):
+        get_json_data_mock.return_value = {
+            'role': 'langpack',
+            'languages-provided': {
+                'es': {},
+                'de': {},
+            },
+            'languages-target': {
+                'app://*.gaiamobile.org/manifest.webapp': '2.2'
+            }
+        }
+        expected = [u'Your language pack contains too many languages. '
+                    u'Only one language per pack is supported.']
```

Comment:
I wonder if these error messages could be a variable that is pulled from the same place so we don't have to keep them in sync if they change?